tests/unit/crypto/test_keccak_sanity.py

```diff
@@ -37,15 +37,15 @@ def test_secure_random_range(self):
     def test_keccak_digest(self):
         data = b'this is a test'
 
-        digest1 = sha3.keccak_256(data).digest()
```

Comment:
Does it make sense to store some well-known output from the `sha3` implementation as constants and then use for comparison, to confirm/reassure we are still backwards-compatible?

Reply:
...I thought we decided we didn't care, and that we were OK with obseleting legacy fingerprints?

Reply:
This test is only changing the library we use to compute the keccak_256 digest, no?  If so, we still compute the keccak digest so I was wondering if we want to ensure that the new library produces the same digest as the old library. Or is that excessive and/or irrelevant now?

Reply:
Yep, I think it's okay to obsolete old fingerprints.